src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
```

Comment:
Random must be a class field

Reply:
Example: 
![image](https://user-images.githubusercontent.com/12864128/220127841-0f638c01-0ccf-4daa-84a6-383703ea42f9.png)

Reply:
not fixed